mtp_prisoner_location_admin/settings/base.py

```diff
@@ -23,6 +31,8 @@
 # SECURITY WARNING: don't run with debug turned on in production!
 DEBUG = True
 
+TEMPLATE_DEBUG = True
```

Comment:
Do we override this in the production settings? Oh, apparently we don't even do that for `DEBUG`. OK, well, we should probably have a task next sprint for setting up production configs.